pkg/render/node_test.go

```diff
@@ -57,6 +57,7 @@ var _ = Describe("Node rendering tests", func() {
 	})
 
 	It("should render all resources for a default configuration", func() {
+		defaultInstance.Spec.FlexVolumePath = "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/"
```

Comment:
Might be nicer to just set this in the `BeforeEach` so that we don't need to do it in every It()?

Reply:
It's a different string depending on the platform

Reply:
Yeah, it defaults to a different string depending on the platform (though it can always be overridden by user config). I guess it's fine, but it confused me a bit.